noir-projects/aztec-nr/aztec/src/state_vars/shared_mutable/scheduled_delay_change/test.nr

```diff
@@ -2,76 +2,6 @@ use crate::state_vars::shared_mutable::scheduled_delay_change::ScheduledDelayCha
 
 global TEST_INITIAL_DELAY: u32 = 13;
 
-unconstrained fn assert_equal_after_conversion(original: ScheduledDelayChange<TEST_INITIAL_DELAY>) {
```

Comment:
sdc now doesn't implement Packable so I had to delete these tests. They were moved with modifications to tests of SharedMutableValues.